src/components/ProfilePage.js

```diff
@@ -202,7 +203,8 @@ export default function ProfilePage() {
             </div>
                 )
             }                   
-        </Container>      
+        </Container>     
+        {id !== username && <Rating firstName={firstName} userId={id} /> }
```

Comment:
Use `canReview` from the GET `/users/:id` endpoint instead of `id !== username` since not every visiting user can edit the visited user (have the users worked together in the past, has the visiting user already rated the user, etc.)